### DISCUSSION TITLE
Is it possible to have a MM2 fan-out setup with strimzi?

### DISCUSSION THREAD

**Discussion by iagotomas** [strimzi/strimzi-kafka-operator#6390] [Q&A] [answered]
Hi,
we are looking into having a fan-out setup managed with Strimzi, in which we would like to have a fan-out solution with a single Kafka Connect cluster using the "source" kafka cluster and producing into multiple target environments. As far as I understood this is not possible with Strimzi, nor with Kafka Connect MirrorSourceConnector. But could this be possible with MM2? 

Thanks for any help,

**Comment by scholzj** [comment, marked as answer]
I do not think that is possible. MM2 used as Connect connectors supports currently only multiple source clusters and one target cluster AFAIK. So to achieve the fanout, you would need to deploy multiple separate MM2 instances.

**Reply by iagotomas** [reply]
Thank you for confirming, we were getting mad trying it out. :) 
By the way, great product!
